### PR TITLE
Add examples to schemas 14 and 15

### PR DESCRIPTION
**Pull request by dbrgn**

### CODE REVIEW

File: 15-draft.json [hunk 1]
```diff
@@ -1151,7 +1209,12 @@
           "properties": {
             "type": {
               "description": "Type of the feed, for example <samp>rss</samp>, <samp>atom</atom>, <samp>ical</samp>",
```
Comment (rnestler) [inline 15-draft.json:1211]:
```suggestion
              "description": "Type of the feed",
```

Remove the "for example" here?

File: 15-draft.json [hunk 2]
```diff
@@ -1134,7 +1187,12 @@
...
```
Comment (rnestler) [inline 15-draft.json:1189]:
```suggestion
              "description": "Type of the feed.",
```

File: 15-draft.json [hunk 3]
```diff
@@ -1117,7 +1165,12 @@
...
```
Comment (rnestler) [inline 15-draft.json:1167]:
```suggestion
              "description": "Type of the feed",
```

File: 15-draft.json [hunk 4]
```diff
@@ -1100,7 +1143,12 @@
...
```
Comment (rnestler) [inline 15-draft.json:1145]:
```suggestion
              "description": "Type of the feed",
```

File: 15-draft.json [hunk 5]
```diff
@@ -176,43 +187,65 @@
...
-          "type": "string"
+          "type": "string",
+          "examples": [
+            "irc://example.org/#channelname"
+          ]
         },
         "twitter": {
-          "description": "Twitter handle, with leading @",
-          "type": "string"
+          "description": "Twitter username with leading <samp>@</samp>.",
```
Comment (rnestler) [inline 15-draft.json:237]: Why the `<samp>` tag here?

File: 15-draft.json [hunk 6]
```diff
@@ -176,43 +187,65 @@
...
+                "type": "string",
+                "examples": [
+                  "@user:example.org"
+                ]
               }
             }
           }
         },
         "irc": {
           "description": "URL of the IRC channel, in the form <samp>irc://example.org/#channelname</samp>",
```
Comment (rnestler) [inline 15-draft.json:230]:
```suggestion
          "description": "URL of the IRC channel",
```

File: 15-draft.json [hunk 7]
```diff
@@ -176,43 +187,65 @@
...
+                  "+1 800 555 4567",
+                  "+41 79 123 45 67"
+                ]
               },
               "email": {
                 "description": "Email address which can be base64 encoded.",
                 "type": "string"
               },
               "twitter": {
                 "description": "Twitter username with leading <samp>@</samp>.",
```
Comment (rnestler) [inline 15-draft.json:202]: Why the `<samp>` tag here?

File: 14.json [hunk 8]
```diff
@@ -1090,7 +1148,12 @@
           "properties": {
             "type": {
               "description": "Type of the feed, for example <samp>rss</samp>, <samp>atom</atom>, <samp>ical</samp>",
```
Comment (rnestler) [inline 14.json:1150]:
```suggestion
              "description": "Type of the feed",
```

File: 14.json [hunk 9]
```diff
@@ -1073,7 +1126,12 @@
...
```
Comment (rnestler) [inline 14.json:1128]:
```suggestion
              "description": "Type of the feed",
```

File: 14.json [hunk 10]
```diff
@@ -1056,7 +1104,12 @@
...
```
Comment (rnestler) [inline 14.json:1106]:
```suggestion
              "description": "Type of the feed",
```

File: 14.json [hunk 11]
```diff
@@ -1039,7 +1082,12 @@
...
```
Comment (rnestler) [inline 14.json:1084]:
```suggestion
              "description": "Type of the feed",
```

File: 14.json [hunk 12]
```diff
@@ -179,43 +190,65 @@
...
+                "type": "string",
+                "examples": [
+                  "@user:example.org"
+                ]
               }
             }
           }
         },
         "irc": {
           "description": "URL of the IRC channel, in the form <samp>irc://example.org/#channelname</samp>",
```
Comment (rnestler) [inline 14.json:233]:
```suggestion
          "description": "URL of the IRC channel",
```

File: 14.json [hunk 13]
```diff
@@ -179,43 +190,65 @@
...
-          "type": "string"
+          "type": "string",
+          "examples": [
+            "irc://example.org/#channelname"
+          ]
         },
         "twitter": {
-          "description": "Twitter handle, with leading @",
-          "type": "string"
+          "description": "Twitter username with leading <samp>@</samp>.",
```
Comment (rnestler) [inline 14.json:240]: Why the `samp` tag here?

**Review by rnestler — CHANGES_REQUESTED** [review]

**Comment by rnestler** [comment]
On another note: Should we change the feed strings to an enum? Also is it possible to define custom types in json schema and then reference them?

It's pretty duplicated right now.

File: 14.json (hunk 13 above)
Comment (dbrgn) [reply 14.json:240]: Copy-paste from v15...

File: 15-draft.json (hunk 7 above)
Comment (dbrgn) [reply 15-draft.json:202]: I guess for rendering as code. Semantically it should not be `<samp>`, I can fix that.

**Comment by dbrgn** [comment]
@rnestler updated. I also changed some more formatting. If desired, I can extract that commit to a separate PR (but I don't think it's needed, since some of it is related to the newly introduced examples).

> On another note: Should we change the feed strings to an enum?

That's a breaking change that should be discussed separately. I personally don't think it's worth breaking compatibility for a very small gain.

> Also is it possible to define custom types in json schema and then reference them?

No.

**Review by rnestler — APPROVED** [review]

**Review by s3lph — APPROVED** [review]

**Pull request merged**